src/scripts/h5p-multi-media-choice-option.js

```diff
@@ -109,6 +190,105 @@ export class MultiMediaChoiceOption {
     return image;
   }
 
+  /**
+   *  Creates a modal containing a video player 
+   *  
+   */
+  createVideoPlayer(lastFocus) {
+    const modal = document.createElement('div');
+    const modalContainer = document.createElement('div');
+    const modalContent = document.createElement('div');
+    const closeButton = document.createElement('button');
+    const cross = document.createElement('span');
+
+    modal.classList.add('h5p-multi-media-content-media-video-modal');
+    modalContainer.classList.add('h5p-multi-media-content-media-video-modal-container');
+    modalContent.classList.add('h5p-multi-media-content-media-video-modal-content');
+    closeButton.classList.add('h5p-multi-media-content-media-video-close');
+    cross.classList.add('icon-cross');
+    cross.textContent = 'X';
+    modal.setAttribute('aria-modal', 'true');
+    closeButton.setAttribute('aria-label', this.closeModalText);
+
+    modal.appendChild(modalContainer);
+    modalContainer.appendChild(modalContent);
+    modalContent.appendChild(closeButton);
+    closeButton.appendChild(cross);
+    this.frame.appendChild(modal);
+    modal.style.display = 'flex';
```

Comment:
We would always prefer to add style in sass when possible. Looks like you added a class to the modal at l204. I think you should move the display flex away in sass